### DISCUSSION TITLE
Set voice messages as played if the user open them

### DISCUSSION THREAD

**Discussion by fulalas** [d99kris/nchat#203] [Ideas]
Sometimes it can be helpful to check if we listened to a given voice message (using the official client), however when opening a voice message in nchat the tiny dot (that in the official client indicates that it hasn't been played) isn't removed. Also, the other person doesn't know if their voice message has been listened.

A big plus would be having an equivalent of this tiny dot in nchat :)

**Comment by d99kris** [comment]
Thanks for suggesting a feature. I have to admit I was not aware of this functionality in the official clients (this is both Telegram and WhatsApp right?). I agree this would be a useful functionality to support, but not sure when I'll get a chance to add it. Will update here if/when it happens.

**Comment by d99kris** [comment]
Although this current feature request was reported earlier, we'll track this request under #574 which also provides a preliminary patch for Telegram and WhatsApp. I'll proceed to close this current request.
